data/json/npcs/godco/classes.json

```diff
@@ -377,19 +377,14 @@
     "job_description": "I'm just trying to survive.",
     "common": false,
     "bonus_str": { "rng": [ -1, 1 ] },
-    "bonus_dex": { "rng": [ -2, 0 ] },
+    "bonus_dex": { "rng": [ 0, 2 ] },
     "bonus_int": { "rng": [ 0, 2 ] },
     "bonus_per": { "rng": [ -1, 1 ] },
     "worn_override": "GODCO_corrie_worn",
     "carry_override": "GODCO_corrie_carried",
     "weapon_override": "GODCO_corrie_wield",
     "proficiencies": [ "prof_plasticworking", "prof_elec_soldering", "prof_welding_basic" ],
-    "traits": [
-      { "group": "NPC_starting_traits" },
-      { "group": "Appearance_demographics" },
-      { "trait": "NO_BASH" },
-      { "trait": "RETURN_TO_START_POS" }
-    ],
+    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" }, { "trait": "NO_BASH" } ],
```

Comment:
You probably want to remove NO_bash at minimum during the follower dialogue as NO_bash in followers causes additional bugs.

Reply:
Didn't know that! Thanks! I will do that then.